### ISSUE TITLE
Runtime install has missing dependency

### ISSUE THREAD

**Issue by kennykerr** [microsoft/WindowsAppSDK#1012]
I tried installing the runtime from here:

https://github.com/microsoft/WindowsAppSDK/releases/tag/v0.8-preview

It works on a dev box but fails on a machine without dev tools installed:

```
---------------------------
ProjectReunion-0.8Preview-Install-x64.exe - System Error
---------------------------
The code execution cannot proceed because MSVCP140.dll was not found. Reinstalling the program may fix this problem.
---------------------------
OK  
---------------------------
```

**Comment by DrusTheAxe** [comment]
I suspect the installer is linked with the VS CRT which isn't installed on the machine. It should be linked with the Hybrid CRT (ucrt+static). I'll throw up a PR

I bet if you [install the VS redistributable](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0) the installer will run fine.

**Comment by zaryaf** [comment]
@DrusTheAxe @dkbennett  Is this still an ongoing issue or can we close?

**Comment by DrusTheAxe** [comment]
> @DrusTheAxe @dkbennett Is this still an ongoing issue or can we close?

We can close this issue.

This is a dupe of the 'Use the Hybrid CRT' issue/plan. The WindowsAppSDK repository (this one) was updated but not all of the WinAppSDK components have yet (WinUI and DWriteCore come to mind). I'm working with those components/devs so we expect to be 100% HybridCRT for 1.0 (but given schedules and vacations I don't expect it for 1.0-experimental1).

There's work in other repositories (e.g. WinUI) which is tracked but nothing to do in Foundation so we can close this issue.

**Comment by andrewleader** [comment]
@DrusTheAxe is this fixed in 1.0?

**Comment by DrusTheAxe** [comment]
> @DrusTheAxe is this fixed in 1.0?

Yes fixed a while back just forgot to update this issue
